templates/repo/commits_table.tmpl

```diff
@@ -29,8 +29,8 @@
 	</div>
 {{end}}
 
-{{if and .Commits (gt .CommitCount 0)}}
-	{{template "repo/commits_list" .}}
```

Comment:
Still the question, it seems that "wiki" is still using the `repo/commits_list`, why not unify them to the new layout? Is there any strong reason that wiki must have a different layout?